src/cli/Cli.ts

```diff
@@ -448,6 +448,11 @@ export class Cli {
       logStatementType = typeof logStatement;
     }
 
+    // overwrite output value with default value if not explicitly specified
+    if (options.output === undefined) {
```

Comment:
I think we should do this earlier in the process, just before we pass args into commands, in case some commands need to know the output mode.

Reply:
I thought about this the same. I looked in the methods calling this function. These are all static methods too. Is it ok to move it there?

Reply:
I'd check if we could put it just before this line https://github.com/pnp/cli-microsoft365/blob/713002b84c3088c7b69a563a51a08ec05295252f/src/cli/Cli.ts#L179. It's after all validation rules have passed, so we won't be resolving options for nothing, and just before we pass the args to commands. We should be able to get in there an instance of CLI and the config value. The downside is that it will show up in the telemetry just as if someone explicitly set output on the command. If we don't want that, then we should put it right before this line: https://github.com/pnp/cli-microsoft365/blob/713002b84c3088c7b69a563a51a08ec05295252f/src/Command.ts#L106.

Reply:
Does it matter if we track this information in the telemetry? I think it would make sense to track the option accordingly.

Reply:
Makes sense. If you could update the PR then we'll review it and get merged 👍

Reply:
Done